Breeding-Insight/sgn
# [BI-1222] Fixes to BrAPI lists

# Description
Adds following list support for brapi lists:
- External references
- Create date
- Made searching lists easier by removing "search by owner" requirement. 
- Made some fixes to PUT list
- Converted brapi types (germplasm and observation variables) to breedbase types (accessions and traits) and back. 


Is based on BI-1251, so includes some extra germplasm changes not done in this card. 

# Dependencies
- Should be run with BI-1085 on bi-api and bi-web. 

# Testing
- Import a germplasm file [BI-1251_file_parents.xls](https://github.com/Breeding-Insight/sgn/files/7764322/BI-1251_file_parents.xls)
- Specify unique list name (description is optional)
- Should be successful. Check breedbase lists api for the new list. 
- Import the same germplasm file. 
- Specify a duplicate list name
- An error should be display. 
- Changing the name to a unique name will result in a successful import.
